### PR TITLE
Update changelog/version for v12.

### PR DESCRIPTION
**Pull request by Fizzadar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 ### Unreleased
+
+
+# v12.0
+
+Breaking note: this change passes all non Kubetools specific container config through to the generated K8s container spec. Any invalid/unused data would have previously been ignored will now be passed to K8s and throw an error.
+
+- Pass any non-Kubetools specific container config through to K8s container spec
 - Implicitly create target namespace if it does not exist
 - The `--cleanup` flag will now remove remaining empty namespaces
```
Comment (stroebs) [inline CHANGELOG.md]: Mind updating this for clarity to be more specifically _the target namespace_, not any empty namespaces please?

Comment (Fizzadar) [reply CHANGELOG.md]: https://github.com/EDITD/kubetools/pull/59/commits/8d33d67bc7de311d7909f248ac39aca8cba2afd2

**Review by stroebs — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (stroebs) [reply CHANGELOG.md]: ty @Fizzadar!

**Comment by Fizzadar** [comment]
Thanks for the review @stroebs!

**Pull request merged**